include/odp/api/spec/proto_stats.h

```diff
@@ -71,6 +77,55 @@ typedef struct odp_proto_stats_capa_t {
 	 * when that proto stats id is supported.
 	 */
 	uint64_t stats_mask_supported;
+
+	/** ODP_PROTO_STATS_ID_TX_* flags */
+	struct {
+		/** Packet adjust support for ODP_PROTO_STATS_ID_TX_OCT_COUNT0
+		 *
+		 * By default octet counter is updated using `odp_packet_len()`.  When adjust
```

Comment:
> By default octet counter is updated using `odp_packet_len()`

...at the time of the final packet transmit or drop.

I mean, is it clear that the octet count is the final octet count (after e.g. IPsec transformation) and not odp_packet_len() of some earlier moment?

Reply:
Ack, I fixed the comment in V3. It is the final packet transmit length after transformations.